src/wallet/wallet.h

```diff
@@ -465,7 +465,7 @@ class CWalletTx : public CMerkleTx
     bool IsEquivalentTo(const CWalletTx& tx) const;
 
     bool InMempool() const;
-    bool IsTrusted() const;
+    bool IsTrusted(bool ignoreUnspendable = true) const;
```

Comment:
Not sure what you mean -- most other argument names are written in this style, e.g. redeemScript, mapKeyBirth, etc.